library/Mockery/Expectation.php

```diff
@@ -331,7 +340,7 @@ protected function _matchArg($expected, &$actual)
         if ($expected instanceof \Mockery\Matcher\MatcherAbstract) {
             return $expected->match($actual);
         }
-        if (is_a($expected, '\Hamcrest\Matcher') || is_a($expected, '\Hamcrest_Matcher')) {
+        if ($expected instanceof \Hamcrest\Matcher || $expected instanceof \Hamcrest_Matcher) {
```

Comment:
I like `boy scout rule`, but it actually hurts code reviews when used with combination of PR sending IMO, because it introduces changes not related to PR topic.

You can send another PR with just these changes (`is_a` change to `instanceof`).